src/Core/Cache/Factory/Cache.php

```diff
@@ -92,7 +69,7 @@ public function __construct(BaseURL $baseURL, IManageConfigValues $config, Datab
 	 */
 	public function createDistributed(string $type = null): ICanCache
```

Comment:
This is only used in one place and without the parameter, do we still need it? What about the APCU type check that's never hit?

Reply:
what do you mean? It's used here: https://github.com/friendica/friendica/blob/241a512589ac4deee398d922a2e14caea1639bbc/src/Core/Session/Factory/Session.php#L74

Reply:
Yes, without parameter, does it still make sense to have one?

Reply:
Yes you're right.. I now understand :D